src/screens/LoginScreen.tsx

```diff
@@ -50,6 +52,7 @@ export default function LoginScreen(props: {
             track(metricKeys.AUTHENTICATION_AUTOMATIC);
             props.navigation.navigate(props.mainRoute);
           }
+          setSilentAuthenticationFinished(true);
```

Comment:
With this solution, what happens when login fails, but a user presses the button again? It does not seem like the spinner would ever appear again after first login attempt.

Reply:
Silent authentication only happens once, so the spinner should never re-appear unless the app is restarted. When you press sign out in the settings screen, you navigate back to the login screen, which should not trigger a new mount. Even if it did trigger a new mount in some apps, it wouldn't matter cuz the silent authentication would fail after sign-out, so you wouldn't run into the issue we are fixing here

Reply:
Gotcha 🥇

Reply:
To make it more robust, I could add a `setSilentAuthenticationFinished(false)` before authenticating silently